packages/p2p-v2/src/pages/my-profile/screens/MyProfileBalance/MyProfileBalance.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { useAdvertiserStats, useDevice } from '../../../../hooks';
+import InfoOutlineIcon from '../../../../public/ic-info-outline.svg';
+import { numberToCurrencyText } from '../../../../utils';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import './MyProfileBalance.scss';
+
+const MyProfileBalance = () => {
+    const { data: advertiserInfo, isLoading } = useAdvertiserStats();
+    const { isDesktop } = useDevice();
+
+    if (isLoading || !advertiserInfo) return <h1>Loading...</h1>;
+
+    return (
+        <div className='p2p-v2-my-profile-balance'>
+            <div className='p2p-v2-my-profile-balance__amount'>
+                <div>
+                    <Text color='less-prominent' size={isDesktop ? 'sm' : 'md'} weight='normal'>
```

Comment:
can we remove weight since its already normal by default

Reply:
Noted, I can fix it in the next PR I'm working on right now, cuz I need this profile content for constructing the profile page